exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/explain/NumberingRelWriter.java

```diff
@@ -71,7 +76,7 @@ protected void explain_(
     RelMetadataQuery mq = RelMetadataQuery.instance();
     if (!mq.isVisibleInExplain(rel, detailLevel)) {
       // render children in place of this, at same level
-      explainInputs(inputs);
+      explainInputs(rel);
```

Comment:
I think this change will break the hash join explain if the inputs of HJ are swapped (based on line 68 above).

Reply:
@amansinha100  Thank you for the catch. I overlooked the above statement where it reverse populates the list for a hashjoin which is swapped. I will write a new explainInputs similar to that of correlatePrel and handle this specific case. 

Are you fine in general with the approach apart from the above issue.

Reply:
Did you already consider a potential alternative via the 'explainTerms()' API ?  Some operators implement it..e.g `SingleMergeExchangePrel` implement it and the purpose is to customize what is shown in the Explain plan for that particular relnode.   My thought was this could have been cleaner way by putting the same register/unregister logic you already have here into the `CorrelatePrel.explainTerms(NumberingRelWriter)` method.   That way you don't have to check `instanceof` of certain specific operators in the NumberingRelWriter.

Reply:
@amansinha100  I did consider about making changes to explainTerms, but it is not working for the two unnest nodes with same column name. Hence I went with this approach.

Reply:
Ok, in that case I suppose the different alias names are not accessible from within the unnest operator.